server/auth/store.go

```diff
@@ -200,7 +200,6 @@ type AuthBackend interface {
 	BatchTx() AuthBatchTx
 
 	GetUser(string) *authpb.User
-	GetAllUsers() []*authpb.User
```

Comment:
It seems that we should keep `GetAllUsers()`, and we only need to change the implementation of `(atx *authReadTx) UnsafeGetAllUsers()` as you already did.